acme/dask_helpers.py

```diff
@@ -171,11 +171,14 @@ def esi_cluster_setup(partition="8GBXS",
     # Don't start a new cluster on top of an existing one
     try:
         client = get_client()
-        msg = "{}Found existing parallel computing client {}. Not starting new cluster."
-        customPrint(msg.format(funcName, str(client)))
-        if start_client:
-            return client
-        return client.cluster
+        if len([w["memory_limit"] for w in client.cluster.scheduler_info["workers"].values()]) == 0:
```

Comment:
I put together this little `_count_running_workers` helper which is supposed to do exactly this. Does that work in a situation like the one you're laying out in #47? If yes, we could use it here - if not, we could replace it with your code above. This way we would have one way of checking worker status.

Reply:
I just used this line of code because it's what's used to throw the final error that I get https://github.com/esi-neuroscience/acme/blob/cb02c810e980d7f86be33cf2cd84df1d94ba1029/acme/backend.py#L745. Didn't check anything else sorry!

Reply:
Ahem. Good catch. I think for consistency's sake, that line should probably also use `_count_running_workers`... I'll merge your PR and look into this. Maybe (hopefully) I had a reason for using two ways to check for workers... Thanks again for the quick fix!